netbench/README.md

```diff
@@ -20,8 +20,24 @@ Here are a few examples of questions that netbench aims to answer:
 * How does certificate chain length affect handshake throughput?
 * Is implementation "X" interoperable with implementation "Y" of "Z" protocol?
 
+## Quickstart
+A basic use of netbench is demonstrated in the `run-netbench-request-response.sh` script. This script will
+- compile all necessary netbench utilities
+- generate scenario files
+- execute the `request-response.json` scenario file using `s2n-quic` drivers
+- collect statistics from the drivers using `netbench-collector`
+- generate a report in the `./report` directory
```

Comment:
Can we put it in `target/netbench/report` instead? That way we only have to gitignore a single directory.